### ISSUE TITLE
Saved sort order in Series page isn't restored when the page loads

### ISSUE THREAD

**Issue by wyldphyre** [mcpierce/comixed#506]
**Describe the bug**
When drilling into a series the saved sort order is restored in the dropdown, but the comics aren't sorted by it.

**To Reproduce**
1. Go to `Library\Collection\Series` and click into a series with more than 11 comics (enough to confirm that issue # sorting works)
1. Set the sort order to `Sort by issue #`
1. Confirm the comics are now sorted correctly
1. Go `Home`
1. Navigate back to the same series
1. The sort dropdown will be set to `sort by issue #`
1. The comics will not be sorted by issue number

At this point if the sort order is switch to something else and then back to issue # order, the sort will be performed correctly.

**Expected behavior**
When the series page is loaded the saved sort order should be restored.

**Desktop (please complete the following information):**
 - OS: Max OS X 10.15.5
 - Browser: Safari
 - Version: The latest version of release/0.4.0 (last commit e9ede12)

**Comment by mcpierce** [comment]
@wyldphyre Can you try out the latest changes on develop (from #507) and see if this bug is still an issue? I believe it to be fixed now with the new adaptor code.

**Comment by wyldphyre** [comment]
It's different, but still not right. My series with nearly 300 issues now sorts as follows when it loads and is set to issue # sort order. Sorting it manually away from issue # and back will make it sort correctly, but it's still wrong when it next gets loaded.

10
100
101
.....
109
11
110
.....
119
12
120
etc

**Comment by mcpierce** [comment]
Yeah,  it appears that the sort order isn't getting honored when it's retrieved from the backend, and is defaulting to sorting by date added. I've tried it here, and that's what I keep seeing.

I set the sort order to issue # and see it sort properly. But if I reload the page, the sorting order is falling back to date added, even though it's showing issue # as the order in the dropdown and it's what was retrieved from the backend.

**Comment by mcpierce** [comment]
@wyldphyre This issue is now being tracked [here](https://github.com/comixed/comixed/issues/13) in the new central ComiXed repository.
